pkg/gitrepo/repo.go

```diff
@@ -16,6 +18,8 @@ import (
 	"gopkg.in/src-d/go-git.v4/storage/filesystem"
 )
 
+var tagRegex = regexp.MustCompile(`^v?[0-9]+\.[0-9]+\.[0-9]+`)
```

Comment:
`v` prefix must be here.


```suggestion
// tagRegex is slightly modified version (required "v" prefix) of
// regexp from from https://github.com/semver/semver/issues/232:
var tagRegex = regexp.MustCompile(`^v(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)(-(0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(\.(0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*)?(\+[0-9a-zA-Z-]+(\.[0-9a-zA-Z-]+)*)?$`)
```

Reply:
On the git repo we only want to have strict semver tag e.g. `1.0.0`, we don't want to allow any suffix IMO.

Reply:
OK, then let's just replace `v?` with `v`.

Reply:
Fine with me.